### PR TITLE
Issue #4220: Split and Organize Checkstyle inputs by Test in the blocks package: EmptyBlock

### PR DESCRIPTION
**Pull request by Kietzmann**
#4220

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockSemantic2.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // Test case file for checkstyle.
 // Created: 2001
 ////////////////////////////////////////////////////////////////////////////////
-package com.puppycrawl.tools.checkstyle.checks.blocks;
+package com.puppycrawl.tools.checkstyle.checks.blocks.emptyblock;
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockSemantic2.java]: Please update class name too.

Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockSemantic2.java]: To which name I should update?
InputEmptyBlockSemantic2 or some else?

Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockSemantic2.java]: Class name should be the same as file name, so yes, `InputEmptyBlockSemantic2`.

Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockSemantic2.java]: Fixed

**Pull request merged**